src/main/java/ru/jma/userservice/controller/UserController.java

```diff
@@ -0,0 +1,41 @@
+package ru.jma.userservice.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import ru.jma.userservice.model.User;
+import ru.jma.userservice.service.UserService;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class UserController {
```

Comment:
It is better to do according to naming UserRestController

Reply:
According to information on the Internet, the "rest" prefix in the name of the controller is redundancy

An example of the wrong approach: https://github.com/ReDestroyDeR/ReactiveLearn/blob/master/src/main/java/ru/red/reactivelearn/controller/user/rest/UserRestController.java